### DISCUSSION TITLE
is:open implementation 'com.github.chrisbanes:PhotoView:latest.release.here' is a virus which hacked my AndroidManifest.xlm become merge

### DISCUSSION THREAD

**Discussion by weiguoping2014** [community/community#146636] [Programming Help]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/b1dbe95b-ea2a-4959-b2bb-535ee9bbc21a)

After I imported the external dependency like as following:

dependencies {
    implementation 'com.github.chrisbanes:PhotoView:latest.release.here'
}
the AndroidManifest.xlm changed to be merged. Then I closed the previous app project.

The problem is when I create another new app project or even after uninstall the android studio and reinstall it, the AndroidManifest.xlm of the new APP project always automatic become merged and import a lot of unnecessary external dependencies, but I do not need them and I hope I can get the Andriod Studio default setup.

Could someone help me to recover it? Otherwise I have to reinstall my Windows as if it is with virus and change my windows Registry Editor.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @weiguoping2014!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
